packages/sdk-cli/src/models/localRelay/launch.test.ts

```diff
@@ -1,54 +1,101 @@
-import { Writable } from 'stream';
+import { createWriteStream, promises as fsPromises } from 'fs';
+import { ChildProcess } from 'child_process';
+import { join } from 'path';
+
 import { launch } from './launch';
 
 describe('launch', () => {
-  it('logs output to a log stream', async (done) => {
-    const logOutput = 'test output';
-
-    const nodeArgs = ['-e', `console.log('${logOutput}')`];
-    const subprocess = launch(nodeArgs, 'pipe');
+  let subprocess: ChildProcess;
+  let logFilePath: string;
 
-    if (!subprocess.stdout) {
-      throw new Error(
-        "child_process.spawn()'ed subprocess doesn't have stdout",
+  afterEach(async () => {
+    // 'SIGKILL' is guaranteed to terminate Node.js processes.
+    // https://nodejs.org/api/process.html#signal-events
+    if (!subprocess.kill('SIGKILL')) {
+      console.warn(
+        "Couldn't SIGKILL the subprocess. Most probably it has already exited.",
       );
     }
 
-    const logStream = new Writable({
-      write: (chunk: Buffer) => {
-        const data = chunk.toString();
-        // The default highWaterMark (buffer/chunk size) for streams is somewhere in KBs,
-        // which should be more than enough to get out logOutput in full.
-        expect(data).toMatch(logOutput);
-        subprocess.kill();
-        done();
-      },
+    if (logFilePath) {
+      try {
+        await fsPromises.unlink(logFilePath);
+      } catch (error) {
+        if (error !== 'ENOENT') {
+          console.error(error);
+        }
+      }
+    }
+  });
+
+  it('spawns a process and logs output to a log file', (done) => {
+    const logOutput = 'test output';
+    logFilePath = join(__dirname, './launch-test-output.txt');
```

Comment:
Put this in tmpdir just to be safe?

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/ce0aefa1c87fa79ad086c7f0ae9a974469a464bb
https://github.com/Fitbit/developer-bridge/pull/176/commits/026948b525cd4b37caf3754098ad228d0f7b284c
https://github.com/Fitbit/developer-bridge/pull/176/commits/5aa37c699115d9fa3b28ffad8930575bac36741e